src/medCore/database/medDatabaseController.cpp

```diff
@@ -478,6 +488,20 @@ void medDatabaseController::createStudyTable(void)
 void medDatabaseController::createSeriesTable(void)
 {
     QSqlQuery query(this->database());
+
+    // Get all the information about the table columns
+    query.prepare("PRAGMA table_info(series)");
+    if ( !EXEC_QUERY(query) )
+    {
+        qDebug() << DTK_COLOR_FG_RED << query.lastError() << DTK_NO_COLOR;
+    }
+
+    this->addColumnTableIfNeeded(query, "origin");
+    this->addColumnTableIfNeeded(query, "flipAngle");
+    this->addColumnTableIfNeeded(query, "echoTime");
+    this->addColumnTableIfNeeded(query, "repetitionTime");
+    this->addColumnTableIfNeeded(query, "acquisitionTime");
+
```

Comment:
Did you test if this works on a new MUSIC installation ?

Reply:
On a new computer? No, but i saw that you can't use an old version of MUSIC if you use this DB with additional columns

Reply:
Tested on a computer without database, no problem

Reply:
The call to ```addColumnTableIfNeeded``` is done before the sql CREATE TABLE command. This means that the first time MUSIC is run, it will attempt to add a column to a table that does not exist, and then create the table. The column will be added only when MUSIC is run a second time. I think the column addition code should he added further below.

Reply:
Strangely, i tested again, and if i open for the first time MUSIC with a removed database, and import a data, you can find these new columns in the database

Reply:
Maybe MUSIC calls this function twice, but just to be more logical, could you still move the add column code after the create table code ?